release_zips/release_zips.py

```diff
@@ -1,13 +1,36 @@
 #!/usr/bin/env python3
 
+import argparse
 import os
+import shlex
 import subprocess
 
 from github import Github
 from github import GithubException
 
+# Define an allowlist of allowed git arguments
+ALLOWED_GIT_ARGS = ['--shallow-since']
+
+
+def validate_git_args(extra_args):
```

Comment:
Not sure if this is really necessary... we don't pass untrusted input to the action so perhaps allowing other Git arguments would be fine?